src/main/deltachat/webxdc.ts

```diff
@@ -321,6 +321,10 @@ If you think that's a bug and you need that permission, then please open an issu
       )
     })
 
+    ipcMain.handle('close-all-webxdc', () => {
+      this._closeAll()
+    })
+
```

Comment:
Why is webxdc part of this pr?

Reply:
generally this pr is about moving code to the frontend and calling electron through runtime.ts
This function is ran at logout which I partly already moved to the frontend.